test/testFortranMPAppIDs.f90

```diff
@@ -0,0 +1,86 @@
+module MYDATA_QUEUE
+    include "queue.f90"
+
+    subroutine create_app_ids(queue)
+        type (QUEUE_STRUCT), pointer :: queue
+        logical :: success
+
+        call queue_create(queue, 5)
+        call queue_append_data( queue, 122, success )
+        call queue_append_data( queue, 233, success )
+        call queue_append_data( queue, 344, success )
+        call queue_append_data( queue, 455, success )
+        call queue_append_data( queue, 566, success )
```

Comment:
`success` is not really used/checked.
I prefer to not have success logical out and instead "exit" (in queue.f90) when queue is full and application tries to append data.